src/wp-includes/class-wp-theme-json-schema.php

```diff
@@ -47,8 +48,14 @@ public static function migrate( $theme_json ) {
 			);
 		}
 
-		if ( 1 === $theme_json['version'] ) {
-			$theme_json = self::migrate_v1_to_v2( $theme_json );
+		// Migrate each version in order starting with the current version.
+		switch ( $theme_json['version'] ) {
+			case 1:
+				$theme_json = self::migrate_v1_to_v2( $theme_json );
+				// no break
```

Comment:
I like them! But they should probably follow coding standards 😞

Reply:
I don't think it's a standard, I was just thinking that the comment doesn't add much. There's no break. 😄

Reply:
What if it said `break deliberately omitted`?

Reply:
I think it could help to stop someone adding a break later on.

When I say following coding standards, I mean it should have a capital first letter and end with a period (if the comment is kept).

Reply:
Sounds like it's fine to do in a follow-up?

Reply:
The linter made me add them. It required the exact string `// no break`.

Reply:
Well, I ran PHPCS in wordpress-develop and it doesn't have the same rule as Gutenberg.

```
FILE: /Users/ajlende/Documents/gutenberg/lib/class-wp-theme-json-schema-gutenberg.php
--------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
--------------------------------------------------------------------------------------
 56 | ERROR | There must be a comment when fall-through is intentional in a non-empty
    |       | case body (PSR2.ControlStructures.SwitchDeclaration.TerminatingComment)
--------------------------------------------------------------------------------------
```

Reply:
I did some testing and it looks like it only matters that the line starts with `// no break`. I can add some additional description if that's helpful.

```php
		// Migrate each version in order starting with the current version.
		switch ( $theme_json['version'] ) {
			case 1:
				$theme_json = self::migrate_v1_to_v2( $theme_json );
				// no break. Once migrated to v2, also migrate to v3.
			case 2:
				$theme_json = self::migrate_v2_to_v3( $theme_json );
				// no break. Once migrated to v2, also migrate to future versions.
		}
```